docs/designs/build-page.md

```diff
@@ -0,0 +1,217 @@
+# Build page workflow
```

Comment:
Wondering would it be clearer if we have two separate flows for conceptual and SDP? even we implemented them in the same pipeline.

Reply:
Most of the parts of them are same, you can tell from the below workflow picture, only some additional steps for SDP.

For the steps specified for SDP or conceptual, I add [SDP]/[Conceptual] at the beginning.